packages/@react-native-windows/cli/src/runWindows/utils/msbuildtools.ts

```diff
@@ -104,7 +104,7 @@ export default class MSBuildTools {
     }
 
     if (target === 'build') {
-      args.push('/restore', '/p:RestorePackagesConfig=true');
```

Comment:
library dependencies that are autolinked into the solution can still have packages.config

Reply:
the compat between old apps/modules and new modules/apps is the [only] thing that [most] worries me about this change, so it would be great to ensure we don't break existing code and don't cause yet another ecosystem reset by requiring a project upgrade

Reply:
But if we do ever do another reset, I want to rush in binary distribution 😂